api/README.md

```diff
@@ -1150,3 +1153,69 @@ You know that an upgrade has been staged when the `horti-upgrade` document in Co
 ## POST /api/v1/upgrade/complete
 
 Completes a staged upgrade. Throws a `404` if there is no upgrade in the staged position.
+
+# Hydration
+
+Accepts a JSON array of document uuids and returns fully hydrated documents, in the same order in which they were requested. 
+When documents are not found, an entry with the missing uuid and an error is added instead.  
+Supports both GET and POST. 
+Only allowed for users with "online" roles. 
+
+## GET api/v1/hydration
```

Comment:
The actual route appears to be `api/v1/hydrate` not `hydration`. I think `hydrate` is better so update the docs to match?

Reply:
Yea oops :D Sorry